app/views/candidate_interface/degrees/degree/new_university_degree.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= content_for :browser_title, title_with_error_prefix(t('page_titles.university_degree'), @degree.errors.any?) %>
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+    text: 'Back',
+    href: candidate_interface_details_path,
+  ) %>
+<% end %>
+
+<%= form_with(model: @degree, url: candidate_interface_degree_university_degree_path) do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_radio_buttons_fieldset :university_degree_status, legend: { size: 'l', tag: 'h1' } do %>
+
+      <p class="govuk-body">
+        You need a bachelor’s degree or equivalent to do postgraduate teacher training. You do not need a degree to apply for a <%= govuk_link_to 'teacher degree apprenticeship (TDA)', 'https://getintoteaching.education.gov.uk/train-to-be-a-teacher/teacher-degree-apprenticeships' %>.
```

Comment:
My preference is for the text to be in translation files. We are very inconsistent about it though. But you should definitely use the existing translation for the link `url_teacher_degree_apprenticeship`

Reply:
Done on 2c316f008d